drivers/SmartThings/philips-hue/src/disco.lua

```diff
@@ -166,7 +165,7 @@ discovered_bridge_callback = function(driver, bridge_ip, bridge_id, known_dni_to
 
         local api_key = item.success.username
         local bridge_base_url = "https://" .. bridge_ip
-        local api_instance = HueApi.new_bridge_manager(bridge_base_url, api_key)
+        local api_instance = HueDiscovery.disco_api_instances[bridge_id]
```

Comment:
This is another place where we switched to re-using a connection where we weren't previously.  Are we OK with that?

Reply:
Nope, reverted. Caught it when I was dealing with `luacheck`.